precompiles/xcm-utils/src/tests.rs

```diff
@@ -198,7 +198,7 @@ fn test_executor_transact() {
 
 			precompiles()
 				.prepare_test(CryptoAlith, Precompile1, input)
-				.expect_cost(1100001001)
+				.expect_cost(276106001)
```

Comment:
I am surprised that the storage growth is affecting the cost so much.

Reply:
I am still not sure about this cost decrease, could you check why it decreased so much?

Reply:
Resolved: due to the changes concerning `require_weight_at_most` introduced in XCM v5 by this [PR](https://github.com/paritytech/polkadot-sdk/pull/6228/files#diff-0d977f9ae4d08c25f545377ea1d8e373cafe0f2cfcb8dda36c6dd0b0ac757733L27)